src/EventList.js

```diff
@@ -74,12 +74,13 @@ export function EventList(props) {
     }
 
     return (
-      <a href={event['browser_url']} 
-        className="eventCard" 
-        target="_blank" 
-        key={event['id']} 
-        coord={('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']) ? "" + event['location']['location']['latitude'] + "&" + event['location']['location']['longitude'] : ""} 
-        onMouseEnter={(event) => { props.updatedHover(event['currentTarget'].getAttribute('coord')) }} 
+      <a href={event['browser_url']}
+        className="eventCard"
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
Why noreferrer? Would we not want the ewarren mobilize site to know these inbounds are coming from this event map?